src/gui/MainPane/RoomDelegate.qml

```diff
@@ -63,7 +63,10 @@ HTile {
                 spacing: room.spacing
 
                 TitleLabel {
-                    text: model.display_name || qsTr("Empty room")
+                    text:
+                        model.bookmarked ?
+                        "\u2764 " + model.display_name || qsTr("Empty room"):
+                        model.display_name || qsTr("Empty room")
```

Comment:
```suggestion
                    text:
                        (model.bookmarked ? "\u2764 " : "") +
                        (model.display_name || qsTr("Empty room"))
```
Empty rooms can be bookmarked too. Also note the parentheses, to prevent a room with no display name to show as just `"\u2764 "`.